docs/en/advanced/airspeed_validation.md

```diff
@@ -0,0 +1,128 @@
+
+- Faulty or blocked airspeed sensor (pitot tube).
+- Very low sensor resolution.
+
+### Innovation Check
+
+Compares the estimated true airspeed (TAS) to the predicted GNSS-based airspeed (groundspeed minus windspeed).
+If the difference exceeds a threshold ([ASPD_FS_INNOV](#aspd_fs_innov_table)) for a prolonged period ([ASPD_FS_INTEG](#aspd_fs_integ_table)), the check fails.
+
+This check helps detect cases where the measured airspeed is inconsistent with the aircraft’s predicted kinematic state, often indicating sensor drift, (partial) blockage, or sub-optimal sensor placement on the vehicle.
+
+Common failure causes:
+
+- Faulty or blocked airspeed sensor (_sensor fault_)
+- Poor sensor placement on vehicle (_configuration issue_)
+- Poor GNSS data (_estimator innacuracy_)
+- Inaccurate wind estimate (_estimator innacuracy_)
+
+Relevant parameters: [ASPD_FS_INNOV](#aspd_fs_innov_table), [ASPD_FS_INTEG](#aspd_fs_integ_table)
+
+### Load Factor Check
+
+Checks whether the measured airspeed is physically consistent with the aircraft's current load factor.
+If the measured airspeed is too low to plausibly generate the required lift, the sensor reading is considered invalid.
+
+This check helps detect cases where the airspeed sensor may be under-reading during flight.
+
+Common failure causes:
+
+- Faulty or blocked airspeed sensor (_sensor fault_)
+- Uncalibrated airspeed sensor (_configuration issue_)
+- Poor sensor placement on vehicle (_configuration issue_)
+- Incorrect stall speed configuration (_configuration issue_)
+- The vehicle is stalling (_flight condition_)
+
+Relevant parameters: [FW_AIRSPD_STALL](#fw_airspd_stall_table)
+
+### First Principle Check
+
+Validates whether the measured indicated airspeed (IAS) behavior matches expected aircraft response based on throttle and pitch inputs.
+
+Specifically, when throttle is above trim by at least 5% and the aircraft is pitched downward (below [FW_PSP_OFF](#fw_psp_off_table)), the rate of change of the IAS should be consistent with the aircraft behaviour.
+
+If the airspeed readings increase too slowly, this may indicate that the sensor is not responding correctly to dynamic pressure changes — for example, due to pitot tube icing or (partial) blockage.
+
+Common failure causes:
+
+- Faulty or blocked airspeed sensor (_sensor fault_)
+- Pitot tube icing (_sensor fault_)
+- Excessive drag (e.g. flaps down, landing gear deployed, payload) (_flight condition_)
+- Throttle not producing expected thrust (_mechanical issue_)
+- Incorrect trim or max throttle setting (_configuration issue_)
+
+Relevant parameters: [ASPD_FP_T_WINDOW](#aspd_fp_t_window_table), [FW_PSP_OFF](#fw_psp_off_table), [FW_THR_TRIM](#fw_thr_trim_table), [FW_THR_MAX](#fw_thr_max_table)
+
+## TAS Scale Considerations
+
+The True Airspeed (TAS) scale is a dynamic correction factor used to account for discrepancies between measured Indicated Airspeed (IAS) and the actual airspeed an aircraft experiences in flight.
```

Comment:
Actually it's the CAS (calibrated airspeed) scale that's being estimated. The scale from CAS to TAS is purely based on air pressure and temperature.